src/city/health.c

```diff
@@ -201,6 +203,32 @@ static void cause_disease(int total_people)
     }
 }
 
+static void reduce_sickness_level_in_building(building *b)
+{
+    if (!b->has_plague) {
+            if (b->sickness_level) {
+                int city_global_coverage = city_health(); // reduction
+                int city_hospital_coverage = 100 - city_culture_coverage_hospital(); // malus
+                int population = city_population();
+
+                if (population <= 1000) {
+                    city_hospital_coverage = 0;
+                } else if (population <= 2000) {
+                    city_hospital_coverage = city_hospital_coverage / 2;
```

Comment:
Good idea with providing coverage for small cities, but I think maybe we could avoid a health cliff caused by population. Maybe we could provide some small number of base coverage, and that will result in gradually dropping health in population, instead of everything being fine with 1999 people, and health disaster on 2001. Maybe 750 base coverage?

Reply:
Yeah, I can see the problem once 2000 population is reached. That's why there was a coverage malus /2 between 1000 and 2000 population, to let the player notice that not having any hospital hinders the sickness decrease. But if the player does not care at all, he could have a 100% malus once 2000 is reached.

So yes, a 750 base coverage (starting from 1000 population ?) seems a good compromise.

Reply:
Ok I updated the calculation